variations.py

```diff
@@ -1,14 +1,49 @@
 from copy import deepcopy
 
 from .booking import Unit
+from .booking import dataset_from_crownoutput
 from .booking import dataset_from_artusoutput
 from .utils import Selection
 from .utils import Variation
 
 import logging
 logger = logging.getLogger(__name__)
 
+class ChangeDatasetCrown(Variation):
+    """
+    Variation that with the method create makes a deepcopy of
+    the selections and actions inside the unit passed as argument and substitutes
+    the directory attribute with the according variation.
+
+    Args:
+        name (str): name used to identify the instance of
+            this class
+        variation (str)
+    """
+    def __init__(self,
+            name, variation):
+        Variation.__init__(self, name)
+        self.variation = variation
 
+    def create(self, unit):
+        new_selections = deepcopy(unit.selections)
+        new_actions = deepcopy(unit.actions)
+        if self.variation not in unit.dataset.quantities_per_vars:
+            logger.fatal('Variation {} not found in ntuple'.format(self.variation))
+            raise NameError
+        else:
+            list_of_quants = unit.dataset.quantities_per_vars[self.variation]
+            for quant_i in list_of_quants:
+                for cut_i in range(len(new_selections[0].cuts)):
```

Comment:
Did you try this without the range based `for` loops as well? I am not 100% sure it works, but I think it should. It would make the code look a little nicer I would say.
```suggestion
                for cut in new_selections[0].cuts:
```